src/Microsoft.SqlTools.Hosting/README.md

```diff
@@ -0,0 +1,67 @@
+Hosting project contains the necessary classes for implementing a JSON - RPC server. It also contains the Utility classes for Logging, Parsing command line arguments from the RPC client. 
```

Comment:
It's important to call out that this isn't just a generic JSON-RPC server, but rather a specialized one (language server that communicates with the language client). Please make sure the docs all refer to this correctly.

And I haven't gone through all the docs yet, but if there's anything here specialized even further to the ADS data protocol then make sure to provide links to that as well. 

```suggestion
This library contains the necessary classes for implementing an Azure Data Studio [language server](https://code.visualstudio.com/api/language-extensions/language-server-extension-guide). It also contains the Utility classes for Logging, Parsing command line arguments from the language client. 
```